spec/features/user_groups_spec.rb

```diff
@@ -22,7 +22,10 @@
     org = create(:user_group)
     login_as(admin)
 
-    visit edit_user_group_path(org)
+    visit user_groups_path
+    find("a[aria-label='#{t('links.aria.edit', name: org.name)}']").click
+
+    expect(current_path).to eq(edit_user_group_path(org))
```

Comment:
I think there are multiple user_groups on the page, i was thinking it just checks and make sure the link goes to the right user group